recipe/meta.yaml

```
@@ -1,17 +1,17 @@
 {% set name = "s2n" %}
-{% set version = "1.4.2" %}
+{% set version = "1.3.27" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name|lower }}-split
   version: {{ version }}
 
 source:
   url: https://github.com/awslabs/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 65d452d62119000c8056c569a0fe5bcbd261c464c41c945a8244df2e89855af8
+  sha256: 58dde9979867a3209a19098ffcd748a144b2462ec81edca5baf864313ff4f589
 
 build:
   number: 0
-  skip: true  # [win]
+  skip: true  # [win or s390x]
```

```suggestion
  skip: true  # [win]
```
We also have a request for pyarrow on s390x. Could this be enabled?